### DISCUSSION TITLE
blog/backpropagation

### DISCUSSION THREAD

**Discussion by giscus** [HarryWangATX/website#44] [General]
# blog/backpropagation

Wonder how to backprogate through a complex loss function in a neural network? Read more to find out!

https://www.harrycodes.com/blog/backpropagation
